shopinvader_easy_binding/views/product_template.xml

```diff
@@ -17,7 +17,10 @@
                 >
                     <field name="shopinvader_company_backend_id" invisible="1" />
                     <div name="company_shopinvader_published">
-                        <field name="company_shopinvader_published" />
+                        <field
+                            name="company_shopinvader_published"
+                            context="{'bind_immediately': True}"
```

Comment:
So is this used afterall or not? 🤔 
I'm confused by this comment:

> Sadly setting the ctx key on the flag won't work :/
https://github.com/shopinvader/odoo-shopinvader/pull/1240/files#diff-d5fc7a068caecefbc091f09cf590d3ac34b4526cc38533d540e545dee70f505eR38

Reply:
no, I remove it